src/services/functionAppService.test.ts

```diff
@@ -349,7 +345,7 @@ describe("Function App Service", () => {
       ],
     }
 
-    const expectedUploadUrl = `https://${customApp.enabledHostNames[1]}/api/zipdeploy/`;
+    const expectedUploadUrl = `https://${customApp.enabledHostNames[1]}/api/zipdeploy`;
```

Comment:
nit: rename to `expectedPublishUrl` since it could be confuse with blob storage upload